build.githubactions.ps1

```diff
@@ -55,7 +55,10 @@ param(
     $NuspecFilePath,
 
     [string]
-    $RelativeRepoPath
+    $RelativeRepoPath,
+
+    [string]
+    $Copyright = "Copyright @ " + $((Get-Date).year) + " Ed-Fi Alliance, LLC and Contributors"
```

Comment:
Are you using this new parameter Copyright ?.  we are packing nuget package using nuspec file which has <copyright> tag,So it will get that Copyright in nuget package , Please verify